src/components/ModalWrapper.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Button, Modal } from '@mui/material'
+import Divider from 'components/divider'
+import PageHeader from 'components/PageHeader'
+import { ModalProps } from 'types/custom_types'
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  bgcolor: 'background.paper',
+  border: '1px solid #000',
+  boxShadow: 24,
+  width: '50%',
+  p: 2,
+}
+
+export const ModalWrapper = (props: ModalProps) => {
```

Comment:
Can you move the `ModalProps` definition into this file? Or is there maybe a good reason why it should be in the global types file?